lib/src/utf8.dart

```diff
@@ -21,7 +21,13 @@ extension Utf8Pointer on Pointer<Utf8> {
   ///
   /// The UTF-8 code units of the strings are the non-zero code units up to the
   /// first zero code unit.
+  ///
+  /// Returns 0 if this is a [nullptr].
```

Comment:
I'm not sure if this is semantically correct, neither from a Dart nor a C point of view.

In Dart land you'd do `someNullableString.length` in legacy mode and get a `NoSuchMethodError` thrown.
In Dart land in NNBD mode you'd have to do `someNullableString?.length`.
In no case you'd get `0` as an answer.
So this is not very Dart-like.

In C land `strlen` also does not check for `NULL`/`nullptr` [1].

[1] https://stackoverflow.com/questions/5796103/strlen-not-checking-for-null